### PR TITLE
Updates Inspection ERTs

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Updates the presets used in inspection ERTs.
Also changes the High Command radio headset to have an alternate version with provost comms. That way junior officers from high command aren't automatically given provost communications.

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Updated the inspection ERTs to be more consistent.
add: Changed the High Command radio headset to not have provost comms, and made an alternate version that keeps it.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/gear_presets/uscm_ship.dm [hunk 1]
```diff
@@ -565,6 +565,13 @@
 	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_R_STORE)
 	new_human.equip_to_slot_or_del(new /obj/item/device/binoculars/range(new_human), WEAR_L_HAND)
 
+
+/datum/equipment_preset/uscm_ship/so/inspection // Used by /datum/emergency_call/inspection_hc
+	name = "USCM Inspection - Staff Officer (SO)"
+	role_comm_title = "Insp"
+
+/datum/equipment_preset/uscm_ship/so/inspection/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/highcom(new_human), WEAR_L_EAR)
```
Comment (Drulikar) [inline code/modules/gear_presets/uscm_ship.dm:574]: This change means this is all they get:
image
Just a headset

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Pull request closed without merging**